src/main/java/me/gnat008/perworldinventory/config/SettingsRetriever.java

```diff
@@ -0,0 +1,28 @@
+package me.gnat008.perworldinventory.config;
+
+import com.github.authme.configme.SettingsHolder;
+import com.github.authme.configme.knownproperties.ConfigurationData;
+import com.github.authme.configme.knownproperties.ConfigurationDataBuilder;
+import com.github.authme.configme.properties.Property;
+
+/**
+ * Utility class for retrieving {@link Property} fields from
+ * {@link SettingsHolder} implementations via
+ * reflection.
+ */
+public final class SettingsRetriever {
+
+    private SettingsRetriever() {}
+
+    /**
+     * Builds the configuration data for all property fields in AuthMe {@link SettingsHolder} classes.
+     *
+     * @return The configuration data.
+     */
+    public static ConfigurationData buildConfigurationData() {
+        return ConfigurationDataBuilder.collectData(
+                PwiProperties.class,
+                DatabaseProperties.class
+        );
+    }
```

Comment:
This is OK but you need to decide which way to go here. You currently use settings holder classes in three different ways:
1. provided directly to Settings
1. this new class wrapping what you do in Settings
1. class collector in settings consistency test

Especially 1 and 2 need to be merged together. Advantage of 2. is that you only define your settings holder classes at one point and after that you can reuse it everywhere. In AuthMe this makes sense because we have many tests where we want the ConfigurationData of AuthMe's settings.
3 is OK and I'd recommend to leave it because this way you really get _all_ settings holder classes. If you remove a class from this one here, and you remove all of the class' properties in config.yml, the consistency test will still fail because you forgot to delete the class.
What would probably be even better (and I know we don't do it in Authme, either), is to have this class list here live as a field on the class and to have the settings consistency test check that he didn't find more classes with the class collector.